csharp/src/Ice/OpaqueEndpoint.cs

```diff
@@ -298,10 +293,14 @@ public override IEnumerable<Endpoint> ExpandHost(out Endpoint? publishedEndpoint
             Port = port;
             ValueEncoding = Ice2Definitions.Encoding; // not used
 
-            if (options.TryGetValue("options", out string? value))
+            if (options.TryGetValue("option", out string? value))
```

Comment:
Is it "option" or "options"? The item accessor definition above still uses "options".

Reply:
It should be "option" (singular). Fixed.